layouts/lists.tsx

```diff
@@ -27,7 +27,7 @@ const ListPageLayout = ({
   const paths = router.asPath.split("/");
   const selectedTab = paths[3] ?? "overview";
 
-  const tabList = [{ name: "Overview" }, { name: "Activity" }, { name: "Contributors" }];
+  const tabList = [{ name: "Overview" }, { name: "Activity" }];
```

Comment:
We should probably add a redirect also from `/lists/{listId}/contributors` back to `/lists/{listId}`